### ISSUE TITLE
convert int64 -> int

### ISSUE THREAD

**Issue by AlJohri** [benmaier/binpacking#7]
the resulting output is a dict of int64 numpy objects instead of regular ints. it would be simpler if this can be converted back into a normal int for things like json serialization etc.

happy to submit a pr if you're interested

**Comment by benmaier** [comment]
is this really an issue? I usually serialize with `simplejson` like this:

```
import simplejson as json
import numpy

def _json_default(o):
    if isinstance(o, numpy.int64): return int(o)
    elif isinstance(o, numpy.float64): return float(o)
    raise TypeError

json.dump(data, f, iterable_as_array=True, default=_json_default)
```

On the other hand I don't really see any harm in converting back to Python types so if you wanna do a PR, I'd be happy to merge it

**Comment by benmaier** [comment]
maybe one may check for the type before packing and then convert back to this original type

**Comment by AlJohri** [comment]
It’s just a minor inconvenience, I expected the dict to have ints. Ran a long process which failed at the serialization stage :/

I’ll submit a PR.

**Comment by benmaier** [comment]
Oh no, sorry about that. Yeah, I guess it makes more sense to return the originally used type. Looking forward to your contribution.

**Comment by benmaier** [comment]
fixed in #22 (package does not depend on numpy anymore)
